packages/ui/react/src/components/Primitives/Dropdown/Dropdown.tsx

```diff
@@ -313,6 +313,11 @@ const Dropdown = ({
                         activeBackgroundColor={option.activeBackgroundColor}
                         brokenSequence={option.brokenSequence}
                         onClick={(event) => {
+
+                          if(option.isDisabled) {
```

Comment:
@mateus2033 can you fix the prettier error on CI? Seems like running the command below should be enough

```bash
npx prettier --write packages/ui/react/src/components/Primitives/Dropdown/Dropdown.tsx
```

![image](https://github.com/user-attachments/assets/389fc278-4f9f-422b-ba30-f79a9d65ca49)

Reply:
Suggestion applied.